### ISSUE TITLE
How to clear credential in client side in Azure Machine Learning extension in VS code?

### ISSUE THREAD

**Issue by zyeric** [microsoft/vscode-tools-for-ai#2447]
## Expected Behavior

I have two azure accounts **A** and **B**, both of them have been used to logged my personal computer.

My azure compute instance is submitted by **A**.

When I try to access the instance by VS code + Azure machine learning toolkit, my credential was set to **B** by mistake. As a result, it failed to connect to the instance.

How can I clear the credential in my local device so that I can connect to the resource by **B** correctly?

## Actual Behavior

There is no chance to re-select the credential during the connection process. And the error message is

```text
User B cannot connect to the interactive job xxx as it was submitted by A. Please sign in to the correct account or connect to a job submitted by B.
```

## Steps to Reproduce the Problem

  NA

## Specifications

  - Version: azure machine learning v1.0.0, vscode 1.93.0
  - Platform: windows 11
  - Screen Reader:

**Comment by tbombach** [comment]
Hi @zyeric, I apologize for the issue you are encountering. Do you see account B in the Accounts section (n the bottom left of the VS Code window? If so, could you sign out of that account and sign in with account A?

**Comment by zyeric** [comment]
@tbombach thanks for your reply. I tried to clear all extensions cache (%USERPROFILE%/.vscode/extensions) and all settings (%APPDATA%\Code) and it works (although need to reinstall extensions).
I will try your approach next time I encounter the problem.
